drive_to_gps/src/calculate_twist_server.cpp

```diff
@@ -0,0 +1,92 @@
+#include "calculate_degrees.h"
+#include "drive_to_gps/dist_heading.h"
+#include "drive_to_gps/heading.h"
+#include "geometry_msgs/Twist.h"
+#include "ros/ros.h"
+#include "sensor_msgs/NavSatFix.h"
+#include <actionlib/server/simple_action_server.h>
+#include <drive_to_gps/gps_goalAction.h>
+#include <ros/callback_queue.h>
+
+#define MAX_ERROR 5         // error for distance comparisons in metres
```

Comment:
In c++, you should always use `constexpr` for constants instead of macros. they're typesafe and can be namespaced.